app/src/main/java/com/example/desafio/adapter/MyPageAdapter.kt

```diff
@@ -28,9 +29,14 @@ class MyPageAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
             }
         }
     }
+/*
+    fun getFragmentFavorite(): Fragment {
+
+        return FragmentFavorite()
+    }*/
 
     override fun getCount(): Int {
-        return 6
+        return 5
```

Comment:
seguindo a sugestão abaixo você poderia usar a lista que criou e pegar o size dela 

```suggestion
        return categoriesList.size()
```

Reply:
Feito